### DISCUSSION TITLE
How to log transform 0s? (or how to deal with these 0 values?)

### DISCUSSION THREAD

**Discussion by leungwinnie** [sibylhe/mmm_stan#49] [Q&A]
There are some spend data that I want to include in my model as dependent variables, these however are 0s at some weeks.

Since log cannot transform 0s, does it make sense if I manually replace these failed transformations to 0?

And many thanks for your helpful article!

**Comment by sibylhe** [comment]
Usually, log transformation is x_transformed = log(x+1)
